deploy/docker/cp-dts-status-checker/dts-status-checker.py

```diff
@@ -0,0 +1,139 @@
+dts_file = os.path.join(FS_ROOT, CLOUD_DATA_DIR, dts_path, DTS_STATUS_FILE)
+dts_token = os.getenv('API_TOKEN')
+email_template_file = os.path.join(
+    FS_ROOT, CLOUD_DATA_DIR, dts_path, DTS_EMAIL_TEMPLATE)
+
+
+def map_pipe_dts_output_to_dts_status(raw_dts_status):
+    return {"id": raw_dts_status["id"], "status": raw_dts_status["status"], "heartbeat": raw_dts_status["heartbeat"]}
+
+
+def fetch_current_dts_statuses():
+    get_new_dts_status_command = "pipe dts list -jo"
+    process = subprocess.Popen(
+        get_new_dts_status_command.split(), stdout=subprocess.PIPE)
+    output, error = process.communicate()
+    if process.returncode != 0:
+        logger.error(error)
+        sys.exit('Error with getting data')
+    new_dts_status = json.loads(output)
+    return {dts_status["id"]: dts_status for dts_status in map(map_pipe_dts_output_to_dts_status, new_dts_status)}
+
+
+def get_email_body(email_template_file):
+    body = {}
+    if os.path.exists(email_template_file):
+        with codecs.open(email_template_file,  "r", "utf-8") as html_file:
+            body = html_file.read()
+    else:
+        body = """<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+    <html xmlns="http://www.w3.org/1999/xhtml">
+    <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+        <style>
+            table,
+            td {{
+                border: 1px solid black;
+                border-collapse: collapse;
+                padding: 5px;
+            }}
+        </style>
+    </head>
+    <body>
+        <p>Dear user,<br>
+        *** This is a system generated email, do not reply to this email ***<br>
+        <b>New DTS status was detected:</b><br>
+        {}<br>
+        Best regards, 
+        EPM-AWS-Dev Platform<br>
+    </body>
+    </html>
+        """
+
+    return body
+
+
+def generate_table(dts_to_inform):
+    table = """<table>
+    <tr>
+		<td><b>ID</b></td>
+		<td><b>Status</b></td>
+		<td><b>Last heartbeat</b></td>
+	</tr>"""
+    for status in dts_to_inform:
+        table += "<tr>" + "<td>{}</td>".format(status["id"]) + "<td>{}</td>".format(
+            status["status"]) + "<td>{}</td>".format(status["heartbeat"]) + "</td>"
+
+    table += "</table>"
+    return table
+
+
+run_id = os.getenv('RUN_ID', default='0')
+pipeline_name = os.getenv('PIPELINE_NAME', default='DefaultPipeline')
+runs_root = os.getenv('CP_RUNS_ROOT_DIR', default='/runs')
+run_dir = os.getenv('RUN_DIR', default=os.path.join(
+    runs_root, pipeline_name + '-' + run_id))
+log_dir = os.getenv('LOG_DIR', default=os.path.join(run_dir, 'logs'))
+pipeline_api = os.getenv('API')
+
+api = PipelineAPI(api_url=pipeline_api, log_dir=log_dir)
+
+logger = RunLogger(api=api, run_id=run_id)
+logger = TaskLogger(task='DTSStatusCheck', inner=logger)
+logger = LocalLogger(inner=logger)
+
+new_dts_statuses = fetch_current_dts_statuses()
```

Comment:
lets add here log message that script starts to work: "Starting to check DTS" or something similar